test/parallel/test-fs-read-stream.js

```diff
@@ -42,11 +49,13 @@ file.on('data', function(data) {
 
 
 file.on('end', function(chunk) {
+  assert.strictEqual(file.bytesRead, bytesRead);
   callbacks.end++;
 });
 
 
 file.on('close', function() {
+  assert.strictEqual(file.bytesRead, bytesRead);
```

Comment:
You could also add a line that compares `file.bytesRead` with the actual file size, like this:
```js
assert.strictEqual(file.bytesRead, fs.statSync(fn).size);
```